libV2/config/index.js

```diff
@@ -206,6 +208,10 @@ class Config {
 
         parsedConfig.cacheBackend = _loadFromEnv('CACHE_BACKEND', config.cacheBackend);
 
+        parsedConfig.ingestionSchedule = _loadFromEnv('INGESTION_SCHED', config.ingestionSchedule);
```

Comment:
why not `INGESTION_SCHEDULE` to avoid confusion (not worth saving 3 chars IMHO), same for the others.

Generally speaking, it could be a good idea to prefix all env with `UTAPI_` for proper namespacing, but might be too late now.

Reply:
`_loadFromEnv` prefixes it for you